docs/data-sources/index.md

```diff
@@ -1,17 +1,18 @@
 ---
 layout: default
-title: Connecting to data sources
-nav_order: 3
+title: Data Sources
+permalink: /data-sources
+nav_order: 4
 has_children: true
 has_toc: true
 ---
 
-# Connecting to data sources
+# Data Sources
 
-A data source in GrimoireLab refers to a platform (Github, Slack, Dockerhub
+A data source in GrimoireLab refers to a platform (GitHub, Slack, Dockerhub
 etc...) from where GrimoireLab can collect data in order to provide you with a
 holistic picture of software development projects. Currently GrimoireLab
 supports **30+ different sources** and it keeps on increasing.
 
 In order to connect the data source of your choice to GrimoireLab, first check
-whether it is supported in GrimoireLab.
\ No newline at end of file
+whether it is [supported in GrimoireLab](docs/data-sources/supported).
```

Comment:
Could we use the baseurl stuff here as well or does it work without it ?

Reply:
I tested all the links, all these work without adding the baseurl thing. But, let me add it to make it consistent.